changelogs/fragments/1415-valmod_req_mismatch.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - vmadm - removed unused ``debug`` parameter (https://github.com/ansible-collections/community.general/pull/1415).
```

Comment:
```suggestion
  - vmadm - simplification of code (https://github.com/ansible-collections/community.general/pull/1415).
```

Reply:
oopsie, I was faster.. lemme get the file back